### PR TITLE
Terminologyupdates

### PR DESCRIPTION
**Pull request by AneeshPunathil**
Request for merge.

### CODE REVIEW

File: docs/istio-integration/architecture.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ For detailed information on Istio architecture and different components, see [Is
...
  
-**Pilot** is a gRRPC based xDS server and provides configuration data to proxies. Citrix provides an xDS client called `xDS-adaptor` to communicate with this Istio control plane component for installing Citrix ADCs in Istio service mesh. It acts as a gRPC client to the control plane API server and listens to updates. Based on the updates from the control plane, `xDS-adaptor` generates the equivalent Citrix ADC configuration. Then, it configures the Citrix ADC ingress or proxy device accordingly.
+**Pilot** is a gRRPC based xDS server and provides configuration data to proxies. Citrix provides an xDS client called `xDS-adaptor` to communicate with the Istio control plane for installing Citrix ADCs in Istio service mesh. It acts as a gRPC client to the control plane API server and listens to updates. Based on the updates from the control plane, `xDS-adaptor` generates the equivalent Citrix ADC configuration. Then, it configures the Citrix ADC ingress or proxy device accordingly.
```
Comment (subashd) [inline docs/istio-integration/architecture.md:11]: change gRRPC to gRPC

Comment (AneeshPunathil) [reply docs/istio-integration/architecture.md:11]: Updated gRRPC to gRPC.

**Review by subashd — APPROVED** [review]

File: docs/istio-integration/README.md [hunk 2]
```diff
@@ -79,14 +79,14 @@ The `xDS-adaptor` monitors the folder where a service mesh deploys certificates
 
 ### Observability
 
-When a service is deployed in a service mesh, users may be interested to get insights about the service behavior. Citrix ADC proxy provides a rich set of in-built metrics to provide insights about the service behavior. When Citrix ADC CPX is deployed as a sidecar, these metrics represent the telemetry data for an application. It helps to reduce the burden of application developers by eliminating the need to include a lot of instrumentation code in the application. Instead, the developer can focus on the core application logic.
+When a service is deployed in a service mesh, you are interested to get insights about the service behavior. Citrix ADC proxy provides a rich set of in-built metrics to provide insights about the service behavior. When Citrix ADC CPX is deployed as a sidecar, these metrics represent the telemetry data for an application. It helps to reduce the burden of application developers by eliminating the need to include many instrumentation codes in the application. Instead, the developer can focus on the core application logic.
```
Comment (sreejithgs) [inline docs/istio-integration/README.md:82]: Change to : When a service is deployed in a service mesh, you may be interested to get insights about the service behavior.

Comment (sreejithgs) [inline docs/istio-integration/README.md:82]: It helps to reduce the burden of application developers by eliminating the need to include many instrumentation codes in the application.

Change to: It helps to reduce the burden of application developers by eliminating the need to include a lot of instrumentation code in the application.

File: docs/developer-guide.md [hunk 3]
```diff
@@ -16,11 +16,11 @@ To create the xDS-adaptor container, run the following command:
 
     make docker_build
 
-## Testing the Citrix xDS-adaptor
+## Testing the Citrix ADC xDS-adaptor
```
Comment (dheerajng) [inline docs/developer-guide.md:19]: It is `Citrix xDS-Adaptor` only. Not Citrix ADC xDS Adaptor

**Review by dheerajng — COMMENTED** [review]
Check out comment in developer-guide.md.

**Pull request merged**